evalml/pipelines/components/estimators/regressors/exponential_smoothing_regressor.py

```diff
@@ -133,6 +135,48 @@ def predict(self, X, y=None):
         y_pred.name = None
         return infer_feature_types(y_pred)
 
+    def get_prediction_intervals(
+        self,
+        X: pd.DataFrame,
+        y: pd.Series = None,
+        coverage: List[float] = None,
+    ) -> Dict[str, pd.Series]:
+        """Find the prediction intervals using the fitted ExponentialSmoothingRegressor.
+
+        Args:
+            X (pd.DataFrame): Data of shape [n_samples, n_features].
+            y (pd.Series): Target data. Optional.
+            coverage (List[float]): A list of floats between the values 0 and 1 that the upper and lower bounds of the
+                prediction interval should be calculated for.
+
+        Returns:
+            dict: Prediction intervals, keys are in the format {coverage}_lower or {coverage}_upper.
+        """
+        if coverage is None:
+            coverage = [0.95]
+        X, y = self._manage_woodwork(X, y)
+
+        y_pred = self._component_obj._fitted_forecaster.simulate(
+            nsimulations=X.shape[0],
+            repetitions=400,
+            anchor="end",
+        )
```

Comment:
+1 for the comment, will be useful in the future to know that `nsimulations` and `repetitions` are just in case it needs to be adjusted in the future